### ISSUE TITLE
Configuration of the session cookie path for more granular security

### ISSUE THREAD

**Issue by sgal** [ory/hydra#3473]
### Preflight checklist

- [X] I could not find a solution in the existing issues, docs, nor discussions.
- [X] I agree to follow this project's [Code of Conduct](https://github.com/ory/hydra/blob/master/CODE_OF_CONDUCT.md).
- [X] I have read and am following this repository's [Contribution Guidelines](https://github.com/ory/hydra/blob/master/CONTRIBUTING.md).
- [ ] This issue affects my [Ory Network](https://www.ory.sh/) project.
- [ ] I have joined the [Ory Community Slack](https://slack.ory.sh).
- [ ] I am signed up to the [Ory Security Patch Newsletter](https://ory.us10.list-manage.com/subscribe?u=ffb1a878e4ec6c0ed312a3480&id=f605a41b53).

### Describe your problem

We have an umbrella domain for our OIDC solution in a form of `login.company.com`. Under that domain we have the following systems:
- Ory Hydra, under `/<region/idp/` path
- Custom-built authentication server, under `/<region>/lp/` path
- Authentication Web app, under `/login`, `/consent`, etc. paths
- Authentication UI components + service, under `/<region/account/` path

Above mentioned services are owned by separate teams.

We want to limit the exposure of the Ory session cookie only to Hydra, preventing potential leaks in case of application bugs or any other reasons.

### Describe your ideal solution

We would like to configure the `Path` part of the cookie setting to only allow a certain value.
Currently, it is [hard-coded to `/`](https://github.com/ory/hydra/blob/0aefa7200b1e39acabef642c52882881c8aeecf2/consent/strategy_default.go#L442), which exposes the cookie to all systems under that domain.

So, basically, another configuration field for Hydra, on the same level as `cookie_domain`, with `/` as a default value would do the trick.

### Workarounds or alternatives

Proxying calls to Hydra and re-writing the cookie headers to the correct values.

### Version

2.x
